resources/views/service/dashboard.blade.php

```diff
@@ -7,11 +7,18 @@
         @include('service.includes.sidebar')
 
         <div id="main-content">
-
             <h1>Rosie Admin Dashboard</h1>
 
-            <p>Welcome to the Rosie Admin Dashboard. Use the links in the sidebar to view vouchers, or to add a new batch of vouchers.</p>
-
+            <p>Welcome to the Rosie Admin Dashboard.</p>
+                
+            <p>Use the links in the sidebar to:</p>
+            
+            <ul>
+                <li>View vouchers, or to add a new batch of vouchers</li>
+                <li>View, add and edit workers (using the 'View workers' link)</li>
```

Comment:
do we need the `(using the....)` bit?

Reply:
it's just because there isn't an 'edit workers' button. I can remove it if you want?

Reply:
ok, keep it in.